plugins/rest/auth.go

```diff
@@ -90,11 +94,70 @@ func DefaultRoundTripperClient(t *tls.Config, timeout int64) *http.Client {
 	tr.ResponseHeaderTimeout = time.Duration(timeout) * time.Second
 	tr.TLSClientConfig = t
 
+	/*
+		Code changes from STS team for FPA-1247
+		Changes made: converted error log type to debug to suppress error message posted in every fail
+	*/
+	if ipv4DialerEnabled {
+		tr.DialContext = func(ctx context.Context, network, addr string) (net.Conn, error) {
+			dialer := net.Dialer{}
+			if !strings.Contains(addr, "sts") {
+				// if the URL is not STS, use the original address
+				return dialer.DialContext(ctx, network, addr)
+			}
```

Comment:
out of context here - we will use IPv4 strictly for everything if specified by the consumer